.github/workflows/nightly_check.yml

```diff
@@ -0,0 +1,34 @@
+name: Nightly Regression Test
+on:
+  workflow_dispatch: # run on request (no need for PR)
+  schedule:
+    - cron: "0 0 * * *" # Every day at 12:00 AM
+defaults:
+  run:
+    shell: bash
+jobs:
+  nightly_regression_test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['macos-11', 'ubuntu-20.04', 'windows-2019']
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    name: nightly regression test (${{ matrix.os }}, Python ${{ matrix.python-version }})
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Installing python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Installing dependencies
+        run: |
+          pip install -e '.[default,tf,tfds]' pytest pytest-html
```

Comment:
I think that dev PIP installing in CI cannot detect some packaging problems. We have already experienced such cases. It seems that we fix it in this PR or in the near future. A possible solution would be using `tox` like OTX.

Reply:
I see. Let me try to fix it in next PR. In this PR, I'd like to merger it first to check the overall working well.